website/www/site/content/en/documentation/sdks/java-multi-language-pipelines.md

```diff
@@ -155,6 +158,50 @@ export PYTHON_VERSION=<version>
 The pipeline outputs a file with the results to
 **gs://$OUTPUT_BUCKET/count-00000-of-00001**.
 
+### Run with DirectRunner
+
+> **Note:** Multi-language Pipelines need to use [portable](/roadmap/portability/)
+> runners. Portable DirectRunner is still experimental and does not support all
+> Beam features.
+
+1. Create a Python virtual environment with the latest version of Beam Python SDK installed.
+   Please see [here](/get-started/quickstart-py/) for instructions.
+2. Run the job server for portable DirectRunner (implemented in Python)
+
+```
+export JOB_SERVER_PORT=<port>
+
+python -m apache_beam.runners.portability.local_job_service_main -p $JOB_SERVER_PORT
```

Comment:
Don't know exactly why, but this yields 
```
ModuleNotFoundError: No module named 'apache_beam.portability.api'
```

Running from the file is starting it, though.
```
python python/apache_beam/runners/portability/local_job_service_main.py -p $JOB_SERVER_PORT
```

Reply:
Did you create a virtual env and install Apache Beam Python (pip install apache-beam) ?

Reply:
I had recycled an old virtualenv, so that was likely the issue. `pip install --force-reinstall apache-beam` solved the issue. Thanks!